docs/create_cli_docs.py

```diff
@@ -0,0 +1,19 @@
+import click
+from cli.cli import scl
+from pathlib import Path
+
+# Recursively go through each command's --help and write it to cli.rst
+def recursive_help(cmd, file, parent=None):
+    ctx = click.core.Context(cmd, info_name=cmd.name, parent=parent)
+    line = "=" * len(cmd.name)
+    file.write(line + "\n" + cmd.name + "\n" + line + "\n\n")
```

Comment:
Consider using different levels for subcommands. Currently all levels of commands appear the same size in the built docs and thus the nesting of subcommands is unclear.